Logo/bin/generate-assets.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+wd=`pwd`
+destdir="autogenerated"
+pngwidths="300 600 1200"
+totalassets=6
+
+find . -type f | grep svg >/dev/null || {
+	echo "No suitable files found. Sure we are in the right dir?"
+	return
+}
+
+# Dirty check for opened vectors
+grep -l -E ' d="[^"]*[^zZ]"' *.svg >"$destdir/.temp" &&  (
+	echo "This/those files seems to have open paths!";
+	cat  "$destdir/.temp"
+	echo
+)
+	
+# Loop through SVG files 
+count=0;
+titleerrors="";
+
+for file in *.svg ; do
+	let "count++"
+	asset="${file%.*}"
+	
+	# Check for valid SVG title
+	svgtitle=$(awk 'BEGIN{IGNORECASE=1;FS="<title>|</title>";RS=EOF} {print $2}' $file)
+	if [ "$svgtitle" != "$asset" ]; then
+		titleerrors="${titleerrors}Title mismatch in ${file}, should be ${asset}, not ${svgtitle}\n"
+	fi
+	
+	# Build PNG in their resolutions
+	for width in $pngwidths; do
+		tempfile="${wd}/${destdir}/${asset}-${width}.png"
+		inkout=$(inkscape "${wd}/${file}" --export-width=$width --export-png=$tempfile)
+		height=$(echo $inkout | grep -o -E "exported to [0-9]+ x [0-9]+ pixels" | cut -d " " -f 5)
```

Comment:
Needs error checking: what happens if we don't find a valid height?

Reply:
Did some check.
It can happen only if the SVG is *really* corrupted. 
In this scenario, with `set -e`, inkscape will fail and the script exit with error.

Reply:
I think it's still worth checking. What happens if a future version of inkscape changes the format of this log message?

Being able to debug this issue is also a good reason to leave the inkscape output on the screen.

Reply:
Ok so I tried this and I agree it looks better (cleaner) without the inkscape output. My inkscape also has a misconfigured plugin which puts some extra garbage into the output every time it runs...

I solved the other issue by checking the height and only printing the inkscape output if we couldn't find the height.